src/main/java/uk/gov/hmcts/reform/lrdapi/domain/BuildingLocationStatus.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.hmcts.reform.lrdapi.domain;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+import java.io.Serializable;
+import java.util.Set;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EntityListeners;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+
+@Entity(name = "building_location_status")
+@NoArgsConstructor
+@EntityListeners(AuditingEntityListener.class)
+@Getter
+@Setter
+public class BuildingLocationStatus implements Serializable {
+
+    @Id
+    @Column(name = "building_location_status_id")
+    private String buildingStatusId;
+
+    private String status;
```

Comment:
any reason why we don't have the @Column and @Size annotations for `status`? Also, the columns `welsh_status`, `created_time` and `updated_time` seem to be missing?

Reply:
Hey @ishratt 

Please find my reasons below:

- This is a GET API. So, we don't really have any validations on the data that is being read. Hence, `@Size` is not applied.
- Again, because this is a GET API, and we are only interested in the building status and hence, the other fields in the table are not included in the domain class. I believe, If we are not persisting the domain object, we don't need the fields that are of no significance. 
- Coming to `@Column` annotation, it is technically an optional annotation and there is no real mapping configuration to be applied on the `status` column and hence, I have ignored it. If you think it is not readable, then sure, I am happy to add it.

Reply:
Thanks for the explanation @viswajit-kuna - I understand your point, for the scope of this api this makes sense. My concern was more for when we decide to persist/retrieve this object, which will be needed as part of other APIs, so just thought it would be easier to create everything at once. But if you think doing that would be outside of the scope of this GET api, then I'm okay with it.

Reply:
When working with huge number of objects, the unnecessary fields would only contribute to the memory footprint. I think it is better that we add rest of the fields if and when required.